### ISSUE TITLE
[Similar to GH-3464] Safari Chrome: Missing Visual Focus on Checkout: Delivery Mode

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4324]
Even though **Radio buttons** have visual focus this is not perceivable enough for accessibility purposes, it should be treated as missing visual focus.
![Screen Shot 2019-08-29 at 7 06 59 PM](https://user-images.githubusercontent.com/15113219/63982582-fc63a180-ca90-11e9-9da6-5976773cc1f0.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔ (note: radio button focus is square)
- Firefox: ✔ (note: focus border is huge, radio button focus is square)
- Safari: ❓

**Comment by KateChuen** [comment]
Firefox: OK

Missing these:
- unchecked checkbox on Chrome
- checkboxes on Safari
- Back and Continue buttons on Safari

**Comment by developpeurweb** [comment]
The `epic/accessibility` branch is missing this fix GH-3464 for the Chrome display bug

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

**Comment by KateChuen** [comment]
Tested on dev3.
firefox ✅

chrome & safari: @developpeurweb Can you confirm this please?
we can't navigate to the "unselected checkbox". So for instance, the first method is selected by default.
And there's no way to select the 2nd checkbox if I want to, through tabbing.

**Comment by kacperknapik** [comment]
We cannot select the unchecked checkbox while tabbing. On the other hand, while shift-tabbing, we can only select the unchecked one, and we skip the checked one. A new ticket will be created.
